File: gems/test-miser/examples/php/src/Classifier.php
<?php

declare(strict_types=1);

namespace TestMiserExample;

final class Classifier
{
    public static function classify(int $value): string
    {
        if ($value > 10) {
            return 'high';
        }

        if ($value > 0) {
            return 'positive';
        }

        return 'nonpositive';
    }
}

File: gems/test-miser/examples/swift/Sources/Classifier/Classifier.swift
public func classify(_ value: Int) -> String {
    if value > 10 { return "high" }
    if value > 0 { return "positive" }
    return "nonpositive"
}

public func isHigh(_ value: Int) -> Bool { value > 10 }
public func isNonpositive(_ value: Int) -> Bool { value <= 0 }
